duckduckgo/duckduckgo-privacy-extension
# Update autofill version and bundling

**Reviewer:** @jonathanKingston 

## Description:
Removes the autofill styles from this repo, updates the autofill version and the related grunt scripts.

**⚠️ Warning:** You can test this version of autofill by using `npm link` or using this hash: `e98da8cb9ba7d6fafcc484aeb9347cf92d67e500 `. Once this is approved, we can release the autofill tag and use that in _package.json_.

## Steps to test this PR:
- Compile for both Chrome and Firefox
- Open https://gravida.pro/lets-work-together (this page has strict CSPs)
  - The email field should show Dax
  - Open the tooltip, it should show correctly (check that it's not flashing on the top left of the page)
  - Use personal and private addresses and make sure they are correctly autofilled
